cpp/tests/bitmask/bit_mask_test.cu

```diff
@@ -0,0 +1,325 @@
+/*
```

Comment:
I don't see specific tests of thread-safe vs. unsafe bit setting and getting. I think you should add multi-threading tests (host and device).